src/routers/users.js

```diff
@@ -4,6 +4,8 @@ const { createUserPayloadValidator } = require('../middlewares/validators');
 
 const usersController = new UsersController();
 usersRouter.post('/', createUserPayloadValidator, usersController.createUser);
-usersRouter.get('/:id', usersController.getUser);
+usersRouter.put('/:id', usersController.updateUser);
```

Comment:
We also need middleware "updateUserPayload", because user cannot update all fields